cognee/infrastructure/engine/models/DataPoint.py

```
@@ -40,3 +43,5 @@ def get_embeddable_properties(self, data_point):
     @classmethod
     def get_embeddable_property_names(self, data_point):
         return data_point._metadata["index_fields"] or []
+    def get_embeddable_property_names(self, data_point):
```

Ruff (0.8.2)

Redefinition of unused `get_embeddable_property_names` from line 44

(F811)